src/app/stream-deck/util/packager.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  allItemsSelector,
+  currenciesSelector,
+  profileResponseSelector,
+  vaultSelector,
+} from 'app/inventory/selectors';
+import { AccountCurrency, DimStore } from 'app/inventory/store-types';
+import { getArtifactBonus } from 'app/inventory/stores-helpers';
+import { maxLightItemSet } from 'app/loadout-drawer/auto-loadouts';
+import { getLight } from 'app/loadout-drawer/loadout-utils';
+import { totalPostmasterItems } from 'app/loadout-drawer/postmaster';
+import { d2ManifestSelector } from 'app/manifest/selectors';
+import { getCharacterProgressions } from 'app/progress/selectors';
+import { RootState } from 'app/store/types';
+import { DestinyProfileResponse } from 'bungie-api-ts/destiny2';
+
+function getPostMasterItem(store: DimStore, hash: number) {
+  return store.items.find((it) => it.location.inPostmaster && it.hash === hash)?.amount || 0;
+}
+
+function getCurrency(currencies: AccountCurrency[], hash: number) {
+  return currencies.find((curr: AccountCurrency) => curr.itemHash === hash)?.quantity;
+}
+
+// create the postmaster update data
+function streamDeckPostMasterUpdate(store: DimStore) {
+  return {
```

Comment:
Maybe get all items in the postmaster first (via `findItemsByBucket`) then count by type?

Reply:
right, I'll fix that then